src/SlippiGame.ts

```diff
@@ -143,13 +140,13 @@ export default class SlippiGame {
     return settings;
   }
 
-  getLatestFrame(): FrameEntryType | null {
+  getLatestFrame(): FrameEntryType {
```

Comment:
Why does this no longer support null? It is possible that if this function is called before any data has been received in real-time mode that this function should return null

Reply:
I've reverted back to the original code.